BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemState.swift

```diff
@@ -22,11 +22,11 @@ struct AddEditSendItemState: Equatable, Sendable {
         var navigationTitle: String {
             switch self {
             case .add:
-                Localizations.addSend
+                Localizations.newSend.capitalized
```

Comment:
@matt-livefront is this a bad move here?  The nav bar needs "New Send" but the button on the empty view needs "New send".  This felt nice and simple but wanna double check with a second pair of 👀

Reply:
I'm not sure if this would end up causing any issues with the supported languages. Using `capitalized(with:)` and the user's selected locale (based on `UI.initialLanguageCode`) would at least consider locale differences.

@fedemkr Any thoughts on this?

Reply:
Hey @matt-livefront and @fedemkr, just curious if you guys could re-review this comment?

Reply:
I think if we're going to use `capitalize` we should use `capitalized(with:)` to account for any locale differences. And this needs to consider if the user is overriding their language in settings, so it needs to be based on `UI.initialLanguageCode`.

Reply:
ℹ️  I think we could directly use [localizedCapitalized](https://developer.apple.com/documentation/foundation/nsstring/1414885-localizedcapitalized):

> Returns a capitalized representation of the receiver using the current locale

> A capitalized string is a string with the first character in each word changed to its corresponding uppercase value, and all remaining characters set to their corresponding lowercase values

Reply:
Looks like we don't have translations for "Send" in our localizations files yet but `localizedCapitalized` seems like the right move here imo.

![Simulator Screenshot - iPhone 15 Pro - 2024-10-14 at 11 11 39](https://github.com/user-attachments/assets/6ec69928-4227-4b1a-9666-db0cb1dcf8dc)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-14 at 11 12 13](https://github.com/user-attachments/assets/b1bc7a7d-494d-41c4-b646-7d6df8985e5e)

Reply:
If the user overrides the system language, won't `localizedCapitalized` still capitalize the string based on the system locale, not their overridden language?

Reply:
ah yeah, thats a good point...